Testscripts/Linux/validate-da.sh

```diff
@@ -25,21 +25,20 @@ readonly dir_list=(/opt/microsoft/dependency-agent /var/opt/microsoft/dependency
 
 readonly da_installer="./InstallDependencyAgent-Linux64.bin"
 readonly uninstaller="/opt/microsoft/dependency-agent/uninstall"
+readonly da="/etc/init.d/microsoft-dependency-agent"
 
 readonly da_pid_file="/etc/opt/microsoft/dependency-agent/config/DA_PID"
 readonly da_log_dir="/var/opt/microsoft/dependency-agent/log"
 readonly da_storage_dir="/var/opt/microsoft/dependency-agent/storage"
 
 function test_case_cleanup() {
-    if [ -f $uninstaller ]; then
-        $uninstaller
-        rm -f "$uninstaller"
+    # Stop DA from running when test fails
+    if [ -f $da ]; then
+        $da stop
     fi
 
     rm -f "$da_installer"
```

Comment:
If there was a failure, you may want to leave the uninstaller so that someone can run it to clean up the system when they are done looking at the logs.